src/plugins/visualizations/public/vis.ts

```diff
@@ -43,7 +43,7 @@ export interface SerializedVisData {
 
 export interface SerializedVis<T = VisParams> {
   id?: string;
-  title: string;
+  title?: string;
```

Comment:
When getting a Visualize embeddable by value, you can end up with duplicate titles (one in `explicitInput -> savedVis -> title` and the other in `explicitInput -> title`)  - for example, when unlinking it from the library. To prevent this, I made `explicitInput -> savedVis -> title` optional so that I can remove it when by value - the only title in a by value embeddable should be `explicitInput -> title`